distributed/tests/test_client.py

```diff
@@ -6590,8 +6591,16 @@ async def f(stacklevel, mode=None):
         """
         x = da.random.random((1000, 1000), chunks=(100, 100))
         with tmpfile(extension="html") as fn:
+            urlpath = fn
+            if not local:
+                pytest.importorskip("fsspec")
+                # Make it look like an fsspec path
+                urlpath = f"file://{fn}"
             async with performance_report(
-                filename=fn, stacklevel=stacklevel, mode=mode
+                filename=urlpath,
+                stacklevel=stacklevel,
+                mode=mode,
+                storage_options=local,
```

Comment:
This test seems unhappy about `storage_options=local` since `storage_options` now needs to be a `dict`. We can probably just remove that line altogether

Reply:
I changed it to dict to test this piece of code.